releases/v0.7.2.md

```diff
@@ -0,0 +1,61 @@
+# v0.7.2
+
+Thank you to everyone for trying Vela! This new release contains bug fixes, enhancements to existing features and new features!
+
+### Breaking Changes notes:
+
+Authentication now comes with a configurable expiration (8hr Refresh Tokens with 15m Access Tokens by default). The following scenarios will require you to [create a GitHub Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) and perform the respective actions:
+
+* Using CLI in automation:
+  * Ensure you are using `v0.7.2` or later of the CLI, whether direct binary or docker image, eg. `target/vela-cli:v0.7.2`.
+  * Supply the Personal Access Token that was created via `VELA_TOKEN` environment variable or `--api.token` flag.
+
+* Using the [vela-downstream plugin](https://github.com/go-vela/vela-downstream):
+  * Ensure you are using `v0.4.0` or later of the plugin.
+  * Supply the Personal Access Token that was created via `DOWNSTREAM_TOKEN` environment variable or `token` parameter.
+
+_Note: the Personal Access Token should have the following scopes: `repo`, `repo:status`, `user:email`, `read:user`, and `read:org`_
+
+### Features:
+
+* Cancel a running build via the UI/CLI/API
+* Add time-sensitive auth with expiry. Eight-hour session length refreshed at 15min intervals
+* Support for writing templates in [Starlark](https://github.com/bazelbuild/starlark) (dialect of python) language
+* New Pipeline management endpoints:
+    * Expand templates within the CLI/UI
+    * Test compiling without triggering a pipeline
+* Execute a vela pipeline locally i.e. vela exec
+* Ability to modify clone process within a pipeline
+* View services natively within the UI
+
+### Enhancements:
```

Comment:
Thoughts on adding some basic screenshots to showcase the UI changes?

Reply:
hmm, i think a lot of the PRs had screenshots (if not all of them) - to avoid duplication maybe i could link to those.

Reply:
hmm, some of them are ok, but nothing consistent. something we can get better at. might reserve for future releases.